tox.ini

```diff
@@ -12,7 +12,7 @@ deps =
     flake8-string-format
 
 commands =
-   python3 -m pip install disparity_interpolation==1.0.3
+   python3 -m pip install numba
```

Comment:
thanks for the PR @jhonykaesemodel, this is really solid.

Should we perhaps add `numba` to the list of installation requirements in setup()?

Reply:
Yes, we should make it a requirement in setup.